### DISCUSSION TITLE
Peut-on utiliser publicode "juste" pour du calcul ?

### DISCUSSION THREAD

**Discussion by BeeDi** [publicodes/publicodes#45] [Q&A] [answered]
Bonjour à vous, 
Je me demandais si vous aviez prévu un usage purement calculatoire de publicodes ? 
J'ai toujours l'impression qu'on en vient à utiliser React pour formater le résultat d'un calcul en HTML. 
Je demande car le langage a l'air prometteur pour décrire des règles de calculs, mais s'il force l'expression du résultat dans un langage particulier ça me poserait problème pour l'utiliser dans une API. 
ps : je ne m'y connais pas trop en React, donc c'est peut être une incompréhension de ma part.

**Comment by laem** [comment, marked as answer]
Oui, c'est prévu, voir le point "Séparer le moteur coeur et le rendu de la documentation " dans https://github.com/betagouv/mon-entreprise/issues/1071

Cela dit, en utilisant le moteur côté serveur, le fait qu'il embarque React n'est a priori (sauf calculs massifs) pas trop pénalisant :)
